tests/src/test/scala/rsc/tests/CheckBaseTests.scala

```diff
@@ -0,0 +1,17 @@
+package rsc.tests
+import java.nio.file.Paths
+import org.scalatest.FunSuite
+import rsc.checkbase.SourceDependencies
+
+class CheckBaseTests extends FunSuite {
+  test("parses source dependencies from cli arg") {
+    val input = "A.scala:B.scala,C.scala:D.scala"
+    val result = SourceDependencies
+    /**EndMarker*/.parsePaths(input).parsePaths(input)
```

Comment:
`EndMarker`?

Reply:
Not sure where that came from.  Odd, but will remove.